odbc_fdw.c

```diff
@@ -831,38 +967,38 @@ static bool odbcAnalyzeForeignTable(Relation relation, AcquireSampleRowsFunc *fu
 			(errmsg("----> starting odbcAnalyzeForeignTable")
 			));
 	#endif
-	
+
 	#ifdef DEBUG
 		ereport(NOTICE,
 			(errmsg("----> finishing odbcAnalyzeForeignTable")
 			));
 	#endif
-	
+
 	return false;
 }
 
-static ForeignScan* odbcGetForeignPlan(PlannerInfo *root, RelOptInfo *baserel, 
-	Oid foreigntableid, ForeignPath *best_path, List *tlist, List *scan_clauses)
+static ForeignScan* odbcGetForeignPlan(PlannerInfo *root, RelOptInfo *baserel,
+	Oid foreigntableid, ForeignPath *best_path, List *tlist, List *scan_clauses, Plan *outer_plan)
 {
 	Index scan_relid = baserel->relid;
 	#ifdef DEBUG
 		ereport(NOTICE,
 			(errmsg("----> starting odbcGetForeignPlan")
 			));
 	#endif
-	
+
 	scan_clauses = extract_actual_clauses(scan_clauses, false);
-	
+
 	#ifdef DEBUG
 		ereport(NOTICE,
 			(errmsg("----> finishing odbcGetForeignPlan")
 			));
 	#endif
-	
+
 	return make_foreignscan(tlist, scan_clauses,
                                 scan_relid, NIL, NIL,
                                 NIL /* fdw_scan_tlist */, NIL, /* fdw_recheck_quals */
-                                NIL /* outer_plan */ );
+                                NULL /* outer_plan */ );
```

Comment:
In PostgreSQL NIL is defined as a `NULL` casted to `List*`: http://doxygen.postgresql.org/pg__list_8h_source.html#l00069

So `NULL` could be used anywhere, but `NIL` is more specific.